client/notebooks/Demo_Client_Notebook.ipynb

```diff
@@ -250,26 +256,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 7,
    "id": "01f4365a",
    "metadata": {},
    "outputs": [],
    "source": [
-    "NB_ROWS = 200\n",
+    "NB_ROWS = 100\n",
```

Comment:
Should be 200 or text above should be change to 100

Reply:
nice catch